docs/docs/README.md

```diff
@@ -21,6 +21,89 @@ To use an existing instance of GATE Teamware as a project manager or admin, find
 
 Documentation on deploying your own instance can be found in the [Developer Guide](developerguide).
```

Comment:
@twinkarma and I have been planning on creating a 'Deployment/Installation guide' section to the docs (i.e. `docs/docs/installguide` or similar) - this might be a good time to do that since this PR is overhauling that section anyway.